agenta-backend/agenta_backend/tasks/evaluations.py

```diff
@@ -427,3 +417,23 @@ def get_app_inputs(app_variant_parameters, openapi_parameters) -> List[Dict[str,
         elif param["type"] == "file_url":
             list_inputs.append({"name": param["name"], "type": "file_url"})
     return list_inputs
+
+
+def parse_correct_answers(evaluator_config_db, data_point) -> List[CorrectAnswer]:
+    """
+    Extracts correct answers based on the evaluator configuration.
+    """
+    correct_answers = []
+    correct_answer_keys = evaluator_config_db.settings_values.get("correct_answer_keys")
+
+    if not correct_answer_keys:
+        return []
+
+    # In case one evaluator has multiple correct answers
+    correct_answer_keys_list = [key.strip() for key in correct_answer_keys.split(",")]
```

Comment:
Not sure what't happening weird, but that looks like bad design. It seems for some reason that `correct_answer_keys` (which is named to sound like a list) is actually a string with commas in between. This bad: you invented a new undocumented datastructure, you named it uncorrectly, and put the logic in a weird place. I was very confused when I found the function being called parse. What is it to parse.
I understand you wanted to find a way to send both one correct_answer and a list, but this hack is not the way.
<img width="686" alt="CleanShot 2024-05-16 at 08 56 54@2x" src="https://github.com/Agenta-AI/agenta/assets/4510758/9671f19e-d618-4717-95ad-a2e1ed8225d8">